### ISSUE TITLE
🦄 Ensure UI complies with accessibility standards

### ISSUE THREAD

**Issue by laritakr** [notch8/hyku-community-issues#65]
## Summary

The new UI work should be compliant with accessibility standards ([WCAG 2.1](https://www.w3.org/TR/WCAG21/))
The Title II requirements are for WCAG 2.1 level AA

Identify accessibility issues and create new implementation tickets identifying fixes needed.

## Requirements

- [ ] New ticket(s) with identified issues.

## Accessibility issues
- [ ] ![Image](https://github.com/user-attachments/assets/103228ee-3210-4aef-b602-031aeed1ef49)
- [ ] ![Image](https://github.com/user-attachments/assets/61b2b51f-4fc6-4360-aa56-8d6359794b0a)
- [ ] ![Image](https://github.com/user-attachments/assets/b1d69860-cd4b-4f76-8c8f-042f4df67951)
- [ ] ![Image](https://github.com/user-attachments/assets/8a016efa-ccfd-450d-931d-1c1bd64fb209)
- [ ] ![Image](https://github.com/user-attachments/assets/59c036c0-c7e6-4fe6-8e26-0143f59ece41)
- [ ] ![Image](https://github.com/user-attachments/assets/2708000b-71dc-466e-9212-4b9e174b38f6)
- [ ] ![Image](https://github.com/user-attachments/assets/65239f1e-66eb-4a5c-ba80-7f24dee166d6)
- [ ] ![Image](https://github.com/user-attachments/assets/6dccefb1-cbb2-40ca-bb37-5b070d073549)
- [ ] ![Image](https://github.com/user-attachments/assets/dacdb049-ea67-467c-8a63-51600f8ff65c)
- [ ] ![Image](https://github.com/user-attachments/assets/c2444c2c-3f74-4e64-9a23-217909c3894c)
- [ ] ![Image](https://github.com/user-attachments/assets/9d453a4a-4abe-4623-b460-cc045ab6fe64)
- [ ] ![Image](https://github.com/user-attachments/assets/f862f67b-6405-4a56-aa29-72c3177c875a)
- [ ] ![Image](https://github.com/user-attachments/assets/acbad42f-9a9f-46cd-8ede-69d4fa92e8f2)
- [ ] ![Image](https://github.com/user-attachments/assets/d25a0ff4-17cf-4d8a-ba10-26cc75015c05)
- [ ] ![Image](https://github.com/user-attachments/assets/dbd4ec33-7121-4d8e-aeea-6df7d61f1c8c)
- [ ] ![Image](https://github.com/user-attachments/assets/311923a1-1feb-4ba1-af1f-9c6b5fceb1f6)
- [ ] ![Image](https://github.com/user-attachments/assets/1e72c36b-07e9-4e0c-bbd0-b4e1830418b7)
- [ ] ![Image](https://github.com/user-attachments/assets/a1da6529-692b-4bd7-b1fc-fd5fdd23d031)
- [ ] ![Image](https://github.com/user-attachments/assets/74293343-5407-4fbd-8415-6830f62f2138)
- [ ] ![Image](https://github.com/user-attachments/assets/80a16d74-046e-49ea-a63e-32718d29a791)
- [ ] ![Image](https://github.com/user-attachments/assets/efbb3c17-58a1-463f-8457-72762f809423)
- [ ] ![Image](https://github.com/user-attachments/assets/8f357379-51a8-4cdb-9788-0141c69c46a6)
- [ ] ![Image](https://github.com/user-attachments/assets/37776ef0-552d-4aef-b1a6-97f491830d3b)

**Comment by ShanaLMoore** [comment]
blocked by 
- #73
